config/locales/en.yml

```diff
@@ -72,7 +72,6 @@ en:
   manage_achievements: "Manage achievements"
   achievement_title: "Title"
   achievement_description: "Description"
-  achievement_points: "Points"
```

Comment:
Good spot. I think these actually are for different places so it may be that we should create a non-conflicted key for one, but we can consolidate them into one for now.